library/core/src/result.rs

```diff
@@ -1203,14 +1203,14 @@ impl<T: Copy, E> Result<&T, E> {
     /// # Examples
     ///
     /// ```
-    /// #![feature(result_copied)]
     /// let val = 12;
     /// let x: Result<&i32, i32> = Ok(&val);
     /// assert_eq!(x, Ok(&12));
     /// let copied = x.copied();
     /// assert_eq!(copied, Ok(12));
     /// ```
-    #[unstable(feature = "result_copied", reason = "newly added", issue = "63168")]
+    #[inline]
```

Comment:
Does `#[inline]` have any meaning here when the method is generic anyway?

Reply:
I'm not sure either! Other methods are all annotated `#[inline]` so I decided to follow suit.

Reply:
It doesn't really matter for generic functions indeed. Maybe it would in a future implementation of rustc, but it's hard to imagine. We could possibly make a lint for it if we decide `#[inline]` will never have an effect on generic functions.